omigami/plotting.py

```diff
@@ -0,0 +1,60 @@
+from PIL.PngImagePlugin import PngImageFile
+from pandas import DataFrame
+from rdkit import Chem
+from rdkit.Chem import Draw
+
```

Comment:
Why didn't you create a class to wrap all these functions?

Reply:
I did not make it a class, because there is no need to instantiated a object of the class or is there?
The functions are technically wrapped by the namespace of the script.